### PR TITLE
removing java webstart

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
This removes Java Webstart and moves custom index page to webclient.

To test on server:
- go to https://trout/merge/webstart/ and check if you get 404
- go to https://trout/merge/webclient/ and check if you can log in, application should work as normal
- check if insight icon is not present on the top bar

To test locally:
- set: ``bin/omero config set omero.web.index_template 'webclient/index.html'``
- start OMERO.web and go to http://localhost:4080/ and check if you are redirected to the http://localhost:4080/index/ and the page shows ``This is index template example``. Click on the webclient link. You should be able to log in as normal.

--no-rebase

### CODE REVIEW

File: components/tools/OmeroWeb/omeroweb/webclient/urls.py [hunk 1]
```diff
@@ -35,9 +35,6 @@
     # Home page is the main 'Data' page
     url(r'^$', views.load_template, {'menu': 'userdata'}, name="webindex"),
 
-    # 'Feed' / 'recent'
-    url(r'^feed/$', views.feed, name="web_feed"),
```
Comment (aleksandra-tarkowska) [inline components/tools/OmeroWeb/omeroweb/webclient/urls.py]: any objections to remove these views? I doubt anyone uses them. @chris-allan @will-moore

Comment (will-moore) [inline components/tools/OmeroWeb/omeroweb/webclient/urls.py]: Fine by me. I didn't even remember that exists, and we have no links or docs for it.

Comment (jburel) [inline components/tools/OmeroWeb/omeroweb/webclient/urls.py]: occured => occurred

**Comment by mtbc** [comment]
Local test passes.

**Comment by mtbc** [comment]
404 from https://eel.openmicroscopy.org/breaking/webstart/
404 from https://eel.openmicroscopy.org/breaking/webclient/

**Comment by mtbc** [comment]
404 from OMERO.web from https://trout.openmicroscopy.org/breaking/webstart/
https://trout.openmicroscopy.org/breaking/webclient/ actually has a usable web client with the permissions database installed.

**Comment by aleksandra-tarkowska** [comment]
@mtbc sorry eel breaking doesn;t exist :) dev breaking is on trout

**Comment by mtbc** [comment]
Good to merge.

**Comment by jburel** [comment]
Removed breaking label
Corresponding insight PR gh-4216

**Comment by dominikl** [comment]
Looks good, the tests work like described (on eel instead of trout-breaking now of course). From that point of view: Good to merge (however I can't eligibly review the code changes).

**Comment by will-moore** [comment]
Am I supposed to get 404 at https://trout.openmicroscopy.org/breaking/webstart/ ? I just get a "Webstart not available". No webstart icon when I login to https://trout.openmicroscopy.org/breaking/webclient/

Ah - breaking removed! OK, https://eel.openmicroscopy.org/merge/webstart is 404.
And webclient has no webstart icon.

Code changes look OK. Not tested locally yet...

**Comment by will-moore** [comment]
```omero config set omero.web.index_template 'webclient/index.html' ``` works as expected. Good to merge.

**Comment by aleksandra-tarkowska** [comment]
thanks everyone for testing

**Pull request merged**